pkg/services/apiserver/builder/helper.go

```diff
@@ -23,16 +23,16 @@ import (
 	k8stracing "k8s.io/component-base/tracing"
 	"k8s.io/kube-openapi/pkg/common"
 
-	"github.com/grafana/grafana/pkg/web"
-
 	"github.com/grafana/grafana/pkg/apiserver/endpoints/filters"
 	grafanarest "github.com/grafana/grafana/pkg/apiserver/rest"
 	"github.com/grafana/grafana/pkg/services/apiserver/endpoints/request"
 	"github.com/grafana/grafana/pkg/services/apiserver/options"
 )
 
+type BuildHandlerChainFunc = func(delegateHandler http.Handler, c *genericapiserver.Config) http.Handler
+
 // TODO: this is a temporary hack to make rest.Connecter work with resource level routes
-var pathRewriters = []filters.PathRewriter{
+var PathRewriters = []filters.PathRewriter{
```

Comment:
Needs to be public because its part of DefaultBuildHandlerChain and we would like to copy the code for that, to not overthink injection points in the chain, just let the callsite define them as necessary.